addons/mail/models/mail_followers.py

```diff
@@ -159,7 +160,7 @@ def _get_recipient_data(self, records, message_type, subtype_id, pids=None, cids
         ) %s
 ) AS x
 ORDER BY pid, cid, notif
-""" % ('OR partner.id IN %s' if pids else '', 'OR channel.id IN %s' if cids else '')
+""" % (pids_query if pids else '', 'OR channel.id IN %s' if cids else '')
```

Comment:
The `OR channel.id IN` part should also be modified using the same technique.

Reply:
done